apps/vue-showcase/cypress/e2e/vue-route-guard.cy.ts

```diff
@@ -0,0 +1,66 @@
+describe(`@this-dot/vue-route-guard`, () => {
+  describe(`No authentication set`, () => {
+    before(() => {
+      cy.window().then((win) => {
+        win.sessionStorage.removeItem('VUEROUTEGUARD-TOKEN');
```

Comment:
I can see you use the string `'VUEROUTEGUARD-TOKEN'` several times in this file. That allows for future contributors to maybe add a typo when they extend these tests.

Please extract this string into a constant at the top of the test file and use that constant, so if the token name changes later we need to only modify it at one place

```
const GUARD_TOKEN_NAME = 'VUEROUTEGUARD-TOKEN'

//...

win.sessionStorage.removeItem(GUARD_TOKEN_NAME)
```